app/assets/javascripts/admin/payments/services/stripe_elements.js.coffee

```diff
@@ -29,9 +30,10 @@ angular.module("admin.payments").factory 'AdminStripeElements', ($rootScope, Sta
       @stripe.createPaymentMethod({ type: 'card', card: @card }, @card, cardData).then (response) =>
         if(response.error)
           StatusMessage.display 'error', response.error.message
+          console.log(JSON.stringify(response.error))
```

Comment:
You might as well use [console.error](https://developer.mozilla.org/en-US/docs/Web/API/Console/error) so it'll show the red signal on the dev tools and show the message in red. We won't miss it this way.

Reply:
I change it now. we will get a bugsnag alert :-)